unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -56,6 +57,7 @@ public DCLVideoTexture()
         {
             model = new Model();
 
+            DataStore.i.virtualAudioMixer.sceneSFXVolume.OnChange -= OnVirtualAudioMixerChangedValue;
             DataStore.i.virtualAudioMixer.sceneSFXVolume.OnChange += OnVirtualAudioMixerChangedValue;
```

Comment:
Same concern here. It is a consctructor, which means no subscription of  `OnVirtualAudioMixerChangedValue` of this instance of the class can be for  `DataStore.i.virtualAudioMixer.sceneSFXVolume.OnChange`. Probably again it highlights some problems with an instance not properly being disposed. Can you try to remove it and check if everything works ok without it?